src/natcap/invest/habitat_quality.py

```diff
@@ -213,36 +213,36 @@
         "output": {
             "type": "directory",
             "contents": {
-                "deg_sum_out_c.tif": {
+                "deg_sum_c.tif": {
```

Comment:
I agree that the "out" in these file names is pretty unnecessary. However, in practice we don't like to update intermediate / output names too often because it is something that could affect a users workflow. It's a light backwards incompatible API change essentially. I think it's alright to leave these changes in, but just wanted to mention that!

Reply:
Totally makes sense. The only reason I updated these is because I noticed the output files don't include "out" in their names. So I'm not truly changing the output names, just updating the spec to reflect what the model actually does.